tools/run_danesfield.py

```diff
@@ -219,12 +219,6 @@ def main(args):
 
     gsd = float(config['params'].get('gsd', 0.25))
 
-    #############################################
-    # Run P3D point cloud generation
-    #############################################
-    # This script assumes we already have a pointcloud generated from
-    # Raytheon's P3D.  See the README for information regarding P3D.
-
```

Comment:
I think it makes sense to leave this comment block in.  If we're still supporting P3D in *some* way, this comment provides a bit of context.